specification/hybridconnectivity/HybridConnectivity.Management/tspconfig.yaml

```diff
@@ -21,10 +21,10 @@ options:
     clear-output-folder: true
     examples-dir: "{project-root}/examples"
   "@azure-tools/typespec-java":
+    service-dir: "sdk/hybridconnectivity"
```

Comment:
service-dir is not needed, if it is same as parameters.service-dir.default

Reply:
Fixed by [f173e3a](https://github.com/Azure/azure-rest-api-specs/pull/32243/commits/f173e3aca39dd81f935e78c9baa0669804a7cddc)